src/sentry/static/sentry/app/components/list/symbol/bullet.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+type Props = {
+  className?: string;
+};
+
+const Bullet = ({className}: Props) => <Wrapper className={className} />;
+
+export default Bullet;
+
+const Wrapper = styled('div')`
+  width: 6px;
+  height: 6px;
+  border-radius: 50%;
+  border: 1px solid ${p => p.theme.gray700};
+`;
```

Comment:
This isn't necessary, you can just rename `Wrapper` to `Bullet` and export it

Reply:
sure thing.